documentation/asciidoc/topics/proc_configure_logging.adoc

```diff
@@ -3,7 +3,8 @@
 
 .Procedure
 
-. Modify your custom resource definition to include logging resources and then apply the changes.
+. Specify logging configuration with the `spec.logging` CR object and then
```

Comment:
_~~CR object~~_ or _in your Infinispan CR_. There is nothing like `spec.logging` CR object.